deploy/clowdapp.yaml

```diff
@@ -19,6 +19,47 @@ objects:
   spec:
     envName: ${ENV_NAME}
     deployments:
+    - name: sidekiq
```

Comment:
I'm surprised it isn't a forked process inside the sources-api pod but needs separated deployment

Reply:
Yes it is a dedicated pod with a different entrypoint. Allows for scaling as well.